tests/unit/src/test/scala/tests/imports/FileSuite.scala

```diff
@@ -124,7 +125,7 @@ class FileSuite extends BaseCliSuite {
     includeOutputPath = includeOutputPath,
     onStdout = { stdout =>
       assertNoDiff(
-        stdout,
+        stdout.replace('\\', '/'),
```

Comment:
You can implement this post-processing step in `BaseCliSuite` by overriding `assertNoDiff`. Then we avoid copy-pasting this logic for all new tests.